functions/Backup-DbaDatabase.ps1

```diff
@@ -71,27 +75,47 @@ Backs up AdventureWorks2014 to sql2016's C:\temp folder
 #>
 	[CmdletBinding(DefaultParameterSetName = "Default")]
 	param (
-		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)]
+		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)][parameter(ParameterSetName = "Pipe")]
```

Comment:
Are we always going to get SMO objects?

I've not pushed it yet, but have a version that could be called with:
'fgs','historyrest' | Out-File C:\dbatool\dbs.txt 
Get-Content C:\dbatool\dbs.txt | Backup-DbaDatabase -SqlInstance localhost\sqlexpress2016 -type full -verbose

TBH, wouldn't be an issue to pick the filtering based on the type of input object (like I'm doing with Restore-DbaDatabase)